content/applications/general/auth.rst

```diff
@@ -7,7 +7,7 @@ Authentication
 .. toctree::
    :titlesonly:
 
-   auth/google
+   auth/google_sign_in
```

Comment:
Is this really necessary? We should avoid breaking URLs in versions before `master` unless there is a real need to support it.

Here, it seems that the file is renamed only to align with the branding chosen by Google. If we keep the file name as it is now ('google' in the 'auth' folder), would it confuse the reader?

Reply:
You're right.
I wanted to distinguish "sign in" to avoid confusion with the google spreadsheet doc in the same category... But it turns out it's google spreadsheet that has nothing to do with this category. I'll remove this change right now.

Reply:
@AntoineVDV done